### DISCUSSION TITLE
How to define a button within v9 Dialog to allow Enter key to fire its onClick

### DISCUSSION THREAD

**Discussion by daric81** [microsoft/fluentui#26060] [Q&A] [answered]
I have a dialog with 2 buttons where I want the 2nd button to fire onClick when the 'Enter' key is pressed. It seems the first button that is defined gets this behaviour. I've checked the docs and cant see a way to do this. Am I missing something?

**Comment by smhigley** [comment, marked as answer]
@daric81 the v9 `<Button>` control uses a native `<button>` element, which uses built-in browser behavior to fire a click when enter is pressed. Using it within Dialog or any other control shouldn't change that. If you're seeing something different, could you provide a codesandbox or other example that we could take a look at?
